desktop/settings/linux.md

```diff
@@ -42,6 +42,8 @@ If you choose the integrated terminal, you can run commands in a running contain
 - **Use Docker Compose V2**. Select to enable the `docker-compose` command to
   use Docker Compose V2. For more information, see [Migrate to Compose V2](../../compose/migrate.md).
 
+- **Show CLI hints**. CLI hints and tips when running Docker commands in the CLI.
```

Comment:
Would appreciate if we can stipulate how to disable hints related to https://github.com/docker/compose-cli/pull/2252?

TLDR; "To disable CLI hints, you can set `DOCKER_CLI_HINTS` env var to `false`".

Reply:
Even better if we can add this to the DD 4.21 release notes.

Reply:
Thanks @crazy-max, will get that added in. And I see you already beat me to it on the release notes 👍